sherpa/ui/utils.py

```diff
@@ -5294,7 +5289,7 @@ def list_models(self, show="all"):
         elif show.startswith("1d"):
             select = lambda x: (not x.startswith('xs')) and (not x.endswith("2d"))
         elif show.startswith("2d"):
-            select = lambda x: x.endswith('2d')
+            def select(x): return x.endswith('2d')
```

Comment:
I don't think you meant to do this...

Reply:
Or maybe you did, but it doesn't make sense in this PR.

Reply:
That's black trying to get rid of all lambda functions. It's functionally equivalent, but yes, it was not an intentional change, it's accidentally left over from when I experimented what black would do.